modules/network-observability-deploy-network-policy.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/network-observability-network-policy.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-deploy-network-policy_{context}"]
+= Configuring a network policy by using the FlowCollector custom resource
```

Comment:
Can include ingress in the heading `Configuring a ingress network policy by using ...`